content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -11,25 +10,24 @@ from the system's components must emit [traces](/docs/concepts/signals/traces/),
 [metrics](/docs/concepts/signals/metrics/), and
 [logs](/docs/concepts/signals/logs/).
 
-OpenTelemetry has two primary ways to instrument.
+Using OpenTelemetry, you can instrument your code in two primary ways:
 
-1. [Code-based solutions](/docs/concepts/instrumentation/code-based) via
-   official APIs and SDKs for eleven languages.
-2. [Zero-code solutions](/docs/concepts/instrumentation/zero-code/) that, when
-   installed, instrument libraries you use.
+1. [Zero-code solutions](/docs/concepts/instrumentation/zero-code/)
+2. [Code-based solutions](/docs/concepts/instrumentation/code-based) via
+   official [APIs and SDKs for most languages](/docs/languages/)
 
-Code-based solutions allow you to get rich telemetry from your application
-itself. They let you use the OpenTelemetry API to generate telemetry from your
-application, which acts as an essential complement to the telemetry generated by
-zero-code solutions.
-
-The Zero-code solutions are great for getting started, or when you can't modify
+**Zero-code** solutions are great for getting started, or when you can't modify
 the application you need to get telemetry out of. They provide rich telemetry
 from libraries you use and/or the environment your application runs in. Another
 way to think of it is that they provide information about what's happening _at
 the edges_ of your application.
 
-It's generally recommended that you use both solutions when you can.
+**Code-based** solutions allow you to get deeper insight and rich telemetry from
+your application itself. They let you use the OpenTelemetry API to generate
+telemetry from your application, which acts as an essential complement to the
+telemetry generated by zero-code solutions.
+
+You can use both solutions simultaneously.
```

Comment:
I'm unsure if this was on purpose, but IMHO, it makes more sense to me to mention zero-code first. WDYT @svrnm et al.?

Reply:
this was on purpose yes, we had a discussion on the original issue around that: https://github.com/open-telemetry/opentelemetry.io/pull/3996#discussion_r1493791120

Here is my reasoning:

> I rather prefer leading with code-based and then letting people know that there is zero-code as well, since not all languages have zero-code and for more and more use cases code-based should be what people are doing, that's why I moved that wording around that below and also tried to give a little bit more "when to use what" beyond "start with zero code then move forward to code-based"

Reply:
Thanks for the context. So, would you like for me to revert the order?
Or maybe we can add a note along the lines of what you shared: "not all languages have zero-code and for more and more use cases code-based should be what people are doing"?

Reply:
Let's go back to the original order I think. For now, at least.

Reply:
Original order restored.